### ISSUE TITLE
Update database

### ISSUE THREAD

**Issue by qualitymanifest** [qualitymanifest/fb_railroadfan_scraper#1]
This was written to:
1. Scrape railroadfan
2. Aggregate distinct locations from foambook
3. Merge railroadfan data with distinct foambook data
4. Write out to `info` collection (this should really be renamed, but that's another issue)

The current code just `console.log`s the result rather than writing it. I took out the db write since it seems terribly inefficient to completely rewrite that collection every time this script is run. 

It's been a while so I forgot the specifics but I remember having issues trying to upsert the [merged](https://github.com/qualitymanifest/fb_railroadfan_scraper/blob/649d8b65fd8feab89a3812a055b5363711a56cda/server.js#L110) data. I need to figure out how to upsert it, or some way to diff new `info` with old `info` after step 3 and upsert the changes.
